app/src/main/java/com/example/hodhod/hotels/HotelDetailsActivity.java

```diff
@@ -0,0 +1,63 @@
+package com.example.hodhod.hotels;
+
+import android.os.Bundle;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.hodhod.hotels.models.HotelItem;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class HotelDetailsActivity extends AppCompatActivity {
+
+    private static final String TAG = "HotelDetailsActivity";
+
+    // ui components
+    private ImageView mImageView;
```

Comment:
remove un needed space to avoid conflicts
make comments only above class and functions , variables and function names need to be readable so that no need for extra clarification.